README.md

```diff
@@ -13,6 +13,16 @@
 
 A community repository for Ansible Playbook of OpenSearch Project.
 
+## Version and Branching
+As of now, this helm-charts repository maintains 3 branches:
+* _main_ (Version is 2.x.x for both `os_version` and `os_dashboards_version` in `inventories/opensearch/group_vars/all/all.yml`)
+* _1.x_ (Version is 1.x.x for both `os_version` and `os_dashboards_version` in `inventories/opensearch/group_vars/all/all.yml`)
+<br>
+
```

Comment:
@peterzhuamazon info about 3rd branch `gh-pages` missing, please add

Reply:
There is no gh-pages in ansible-playbook that is why I removed it 😄

Reply:
fixed.